thesaurus/config.py

```diff
@@ -0,0 +1,45 @@
+import os
+
+from wsgiref.simple_server import make_server
+from pyramid.config import Configurator
+
+def configure():
+# does this work? YEEESSSSSS
+    settings = {
+        'pyramid.reload_assets': True,
+        'pyramid.reload_templates': True,
+    }
```

Comment:
Maybe these should only be true in development too?

Reply:
``` python
    settings = {}
    if settings["dache.env"] == Environment.development:
        settings.setdefault("pyramid.reload_assets", True)
        settings.setdefault("pyramid.reload_templates", True)
```

Reply:
👍 That makes sense. I don't think it would cause a slowdown -- it'll only reload things if the files change on disk, right? We wouldn't be changing any files, so it would essentially do nothing, I imagine. Regardless, we should disable it in not-dev mode.